l10n/fr.js

```diff
@@ -9,6 +9,7 @@ OC.L10N.register(
     "Your browser does not support videos." : "Votre navigateur ne prend pas en charge les vidéos.",
     "Open sidebar" : "Ouvrir la barre latérale",
     "Delete" : "Supprimer",
+    "Download" : "Télécharger",
```

Comment:
This is managed by transifex. Please discard it :)

Reply:
I removed it from that commit (hence the foce push)

Reply:
Thank you :).

BTW, sorry, I am totally busy these days and don't have time for finishing this work.

I am very happy other people complete it :).